README.md

```diff
@@ -1,5 +1,11 @@
 # ![nf-validation](docs/images/nf-validation.svg)
 
+> [!IMPORTANT] **`nf-validation` has now been deprecated and won't be updated anymore.**
+>
+> **Please update your pipelines to use [`nf-schema`](https://github.com/nextflow-io/nf-schema) instead.**
+>
+> This change was necessary to prevent older versions of `nf-core` pipelines from breaking when updating to the latest version of `nf-validation`. Some older and current versions of the pipelines did not pin the version of nf-validation, which caused the pipelines to break when the latest version of nf-validation was released. This change will prevent this from happening in the future. **Please pin the version of `nf-schema` in your pipeline's `nextflow.config` file.**
```

Comment:
This [doesn't render properly](https://github.com/nextflow-io/nf-validation/blob/be217ca5447f5eacc72a10c8a0636264320e28cb/README.md). The admonition type needs to be on it's own line.

```suggestion
> [!IMPORTANT]
> **`nf-validation` has now been renamed to `nf-schema`. The `nf-validation` plugin will not receive any future updates.**
> **Please update your pipelines to use [`nf-schema`](https://github.com/nextflow-io/nf-schema) instead.**
>
> This change was necessary to prevent older versions of `nf-core` pipelines from with unpinned plugin references from breaking when updating to the latest version of `nf-validation`.
> **Please pin the version of `nf-schema` in your pipeline's `nextflow.config` file:**
> ```nextflow
> plugins { id 'nf-schema@2.0.0' }
> ```
```